components/Modal/CreateImageUpload.js

```diff
@@ -405,23 +530,46 @@ class CreateImageUploadModal extends React.Component {
       component: (
         <React.Fragment>
           {this.missingRequiredFields() && (
-            <Alert
-              id="required-fields-alert"
-              variant="warning"
-              isInline
-              title={formatMessage(messages.warningMissingFields)}
-            />
+            <Alert id="required-fields-alert" variant="danger" isInline title={formatMessage(messages.warningReview)} />
           )}
           <TextContent>
             <Title className="cc-c-popover__horizontal-group" headingLevel="h3" size="2xl">
               <FormattedMessage defaultMessage="Create and upload image" />
-              <Popover bodyContent={formatMessage(messages.infotip)} aria-label="process length popover">
-                <OutlinedQuestionCircleIcon />
+              <Popover bodyContent={formatMessage(messages.infotip)} aria-label="process length help">
+                <Button variant="plain" aria-label="process length help">
+                  <OutlinedQuestionCircleIcon id="popover-icon" />
+                </Button>
               </Popover>
             </Title>
             <Text>{formatMessage(messages.review)}</Text>
             {providerSettings[uploadService] !== undefined && (
               <TextList component={TextListVariants.dl}>
+                <dt>
+                  <FormattedMessage defaultMessage="Image size" />
+                </dt>
+                <dd>
+                  {imageSize === "" && (
```

Comment:
@jkozol, I tried from UI, `A value is required.` warning message was never got displayed when I made image size input box empty.

Reply:
@henrywang Thanks for fixing this!